bundle.cpp

```diff
@@ -569,6 +571,7 @@ bool ZAppBundle::SignFolder(ZSignAsset* pSignAsset,
 	ZLog::PrintV(">>> BundleVer: \t%s\n", jvRoot["bver"].asCString());
 	ZLog::PrintV(">>> TeamId: \t%s\n", m_pSignAsset->m_strTeamId.c_str());
 	ZLog::PrintV(">>> SubjectCN: \t%s\n", m_pSignAsset->m_strSubjectCN.c_str());
+	ZLog::PrintV(">>> Minimum iOS Version: \t%s\n", jvRoot["ros"].asCString());
```

Comment:
Ditto (see comment above).
```suggestion
	ZLog::PrintV(">>> Minimum OS Version: \t%s\n", jvRoot["ros"].asCString());
```